docs/developer/pandas/partition_api.rst

```diff
@@ -0,0 +1,79 @@
+Pandas Partition API in Modin
+=============================
+
+If you are working with a Modin Dataframe, you can unwrap its remote partitions
+to get the raw future objects compatible with the execution engine (e.g. ``ray.ObjectRef`` for Ray).
+You can use this API to get the IPs of the nodes that hold these objects as well. In that case you can pass the partitions
+having needed IPs to your function. It can help with minimazing of data movement between nodes. In addition to
+unwrapping of the remote partitions we also provide API to construct Modin DataFrame from them.
+
+Ray engine
+----------
+However, it is worth noting that for Modin on ``Ray`` engine with ``pandas`` backend IPs of the remote partitions may not match
```

Comment:
This paragraph should go in the separate section about IPs.

Reply:
Moved.